src/components/scenes/GettingStartedScene.tsx

```diff
@@ -154,7 +154,7 @@ export const GettingStartedScene = (props: Props) => {
   useAnimatedReaction(
     () => swipeOffset.value,
     value => {
-      if (value === 5) {
+      if (value === paginationCount) {
```

Comment:
Shouldn't this have been included in the previous commit? 

Also, how did this work before; I see paginationCount is `section.length + 1`, but that's not included in this commit, nor the previous. Wth?